### ISSUE TITLE
Output of the visualize_distribution() for noise/outlier document

### ISSUE THREAD

**Issue by terilias** [MaartenGr/BERTopic#1424]
Hello, 
First of all thank you for publishing this powerful library! 
My question is about the output barplot of the visualize_distribution() function. I am call it for a document that has been take the topic "-1" but the topic "-1" doesn't shown in the barplot. A different topic seems to have the highest probability instead of the "-1" topic. My thought is that maybe the chart shows the probabilities for the rest of the topics (those that are not "-1") but I didn't read that in the function documentation so I thought I'd write it here....
![Screenshot_20230723_150116](https://github.com/MaartenGr/BERTopic/assets/37142366/fb0bb61b-c5c7-46f1-9f12-5624c9c52109)

**Comment by MaartenGr** [comment]
That's correct! The probabilities of the -1 topic are not directly outputted and should be calculated as 1 - sum(probabilities). As a result, those of "actual" topics/clusters are shown.

**Comment by terilias** [comment]
Thank you for the answer!
